video/mkv/mkvparser.cpp

```diff
@@ -5309,7 +5296,7 @@ long VideoTrack::Parse(Segment* pSegment, const Info& info,
   long long stereo_mode = 0;
 
   double rate = 0.0;
-  std::unique_ptr<char[]> colour_space_ptr;
+  Common::ScopedPtr<char, Common::ArrayDeleter<char>> colour_space_ptr;
```

Comment:
STL, makes me wonder who reviewed this in first place...

Reply:
@aquadran That was me, and moreover, I did add it in the first place...

Reply:
@aquadran That was me, and moreover, I did add it in the first place...